analyzer/src/funTest/kotlin/managers/CarthageFunTest.kt

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2020 Bosch.IO GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.analyzer.managers
+
+import io.kotest.core.spec.style.StringSpec
+import io.kotest.matchers.shouldBe
+
+import java.io.File
+
+import org.ossreviewtoolkit.downloader.VersionControlSystem
+import org.ossreviewtoolkit.utils.normalizeVcsUrl
+import org.ossreviewtoolkit.utils.test.DEFAULT_ANALYZER_CONFIGURATION
+import org.ossreviewtoolkit.utils.test.DEFAULT_REPOSITORY_CONFIGURATION
+import org.ossreviewtoolkit.utils.test.USER_DIR
+import org.ossreviewtoolkit.utils.test.patchExpectedResult
+
+class CarthageFunTest : StringSpec() {
```

Comment:
I believe the current code is not using `Fun` as part of the name of the functional tests.
I also wouldn't see any problem naming it `CarthageTest` even though a file with that name already exists.

Reply:
IntelliJ with Kotest plugin has issues if the tests are named the same.
Starting the funTest from the gutter will run the unit test instead. That's why I opted to name it like that.

That's at least the case with the default created run configuration. I could create multiple run configurations to fix this. But it was pretty confusing that using the gutter icon didn't start the correct test.
What do you think?

Reply:
At least I have never witnessed this issue before, but I'd say it's a must that running the tests from the gutter icons work out of the box. So maybe we should instead consider renaming all functional tests to `*FunTest.kt`? (In a separate PR, of course.)

Reply:
@sschuberth @MarcelBochtler  shall we strip the `Fun` for now for consistency and refactor in a separate PR?

Reply:
I've just created that separate PR at https://github.com/oss-review-toolkit/ort/pull/3111 so we can merge that PR first an then rebase this one on top, so this code can stay as-is.